core/src/main/java/com/amplifyframework/core/DeveloperMenuManager.java

```diff
@@ -52,6 +53,16 @@ public static DeveloperMenuManager sharedInstance(Context context) {
         return sInstance;
     }
 
+    /**
+     * Allows the developer menu to be activated if the app is in a debuggable build.
+     */
+    public void enableDeveloperMenu() {
+        if ((context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE) != 0) {
+            Amplify.Logging.shouldStoreLogs(true);
```

Comment:
I think you still want to strive to have the category not know about log storage. This is leaking your dev menu logic into the category API.

Instead, I would aim to express this as:
```java
Amplify.addPlugin(new PersistentLogStoragePlugin());
```